src/integrated/soil_plant_hydrology_biogeochemistry_model.jl

```diff
@@ -0,0 +1,284 @@
+# Note: as we create more integrated model, it would be good to have a discussion for folder structure and file name convention
```

Comment:
could remove now? unless you wanted to have this option

Reply:
no you're right I'll remove this one, and make one `soil_canopy_biogeochemistry` in another PR